lib/puppet/network/authconfig.rb

```diff
@@ -78,6 +78,12 @@ def mk_acl(acl)
     # raise an Puppet::Network::AuthorizedError if the request
     # is denied.
     def check_authorization(method, path, params)
+      if Puppet.settings[:bypass_authorization] == true
```

Comment:
Happy to go either way, this is what @jeffmccune had in his prototype.

I generally prefer to leave off `== true` but I could see an argument for explicitness.